AUTHORS

```diff
@@ -120,4 +121,3 @@ the authors tag in the respective file header.
  - Witold Wolski
  - Xiao Liang
  - Yasset Perez-Riverol
- - Peter Jones
```

Comment:
Please add Peter back as an author (ideally in the correct alphabetical position)

Reply:
So he is. Good catch.